src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
Chrome has the same formatting issue.  But, only has the mathjax warning
![chrome](https://user-images.githubusercontent.com/28968644/29050444-99037880-7ba1-11e7-907f-1efac2e5ea57.png)